tools/langs/cplusplus.py

```diff
@@ -231,15 +228,14 @@ def __create_mixin_classes(schema, outdir):
         tplvars = {
             "includes": "",
             'license': LICENSE.format(authors=AUTHORS),
-            'moduleNameCaps': "{0}MIXIN".format((module.upper()).replace("-","_")),
+            'moduleNameCaps': "{0}MIXIN".format((module.upper()).replace("-", "_")),
             'elements': classes.strip()
         }
         if "std::string" in classes:
             tplvars["includes"] = "#include <string>"
         fullout = CLASSES_HEAD_TEMPLATE.format(**tplvars)
-        fmh = open(os.path.join(outdir, "{0}mixins.h".format(module.lower())), 'w')
-        fmh.write(fullout)
-        fmh.close()
+        fmh = Path(outdir, "{0}mixins.h".format(module.lower()))
```

Comment:
Should we take the further opportunity to switch to `f` strings?
```suggestion
        fmh = Path(outdir, f"{module.lower()}mixins.h")
```